playkit/src/main/java/com/kaltura/playkit/plugins/Youbora/YouboraLibraryManager.java

```diff
@@ -93,6 +93,7 @@ public void onEvent(PKEvent event) {
                 switch (((PlayerEvent) event).type) {
                     case DURATION_CHANGE:
                         log.d("new duration = " + ((PlayerEvent.DurationChanged) event).duration);
+                        playHandler();
```

Comment:
@AntonAFA this would cause start to be fired very early before user had an intent to actually play.
Need to revert it back to play, and make sure we take into account when play is called and we have ads.
In that case we need to make sure we call start also when ad is played, either by registering to ad started event and calling the playHandler or by any other means that might be relevant.
A more whole approach is to fire adplay when ad decorator is called and register to that on the Youbora plugin, but this is a bigger story.
@noamtamim - FYI